app/src/main/kotlin/com/wire/android/ui/home/conversations/edit/AssetEditMenuItems.kt

```diff
@@ -26,26 +26,47 @@ import com.wire.android.ui.edit.ReactionOption
 import com.wire.android.ui.edit.ReplyMessageOption
 
 @Composable
-fun AssetEditMenuItems(
+fun assetEditMenuItems(
```

Comment:
Composable function name should start with capital letter

Reply:
No, composable functions returning list should not be from Capital letter according to slack Lint https://slackhq.github.io/compose-lints/rules/#naming-composable-functions-properly